docs-src/typescript/chapter-project-setup/project-structure.md

```diff
@@ -0,0 +1,133 @@
+---
+id: project-structure
+title: Boilerplate Temporal Application project code
+sidebar_label: Boilerplate project
+description: Discover the minimum code you  need to create a boilerplate Temporal Application with TypeScript
+tags:
+  - typescript sdk
+  - developer guide
+  - project setup
+---
+
+**What is the minimum code I need to create a boilerplate Temporal Application?**
+
+Let’s start with a single Activity Workflow and register those functions with a Worker.
```

Comment:
Also, since you said "a SINGLE Workflow", then we don't want to say "register THOSE  FUNCTIONS" with a Worker. 

Instead: "Register that function with a Worker."